src/components/TasksTable/index.jsx

```diff
@@ -254,6 +315,43 @@ export default class TasksTable extends Component {
             onFilterClick={this.handleFilterToggle}
           />
         </div>
+        <Drawer
```

Comment:
Clicking on the drawer opens up the drawer with an animation. Clicking on the "X" button closes the drawer without animation. Let's fix that. We want to have the animation applied when the drawer is closed.